### DISCUSSION TITLE
WikiJS error: Cannot read properties of undefined (reading 'id')

### DISCUSSION THREAD

**Discussion by Hagbrain** [requarks/wiki#5231] [Error / Bug Report]
Hello,

were using lastest WIKIJS with Docker after 24h we gettin following error:

```
Oops, something went wrong...
Cannot read properties of undefined (reading 'id')
```

a restart of docker wiki fixing the issue for the next 24h.

**Comment by adamshand** [comment]
Just had the same issue.  Installed Wiki.JS this morning was setting up email and then hid the local authenticate in the settings and got this message.  Restarting the docker container doesn't fix it for me. :-(

**Comment by arkansas-coder** [comment]
I am having the same problem. I've been using WikiJS for awhile with no problems, but I upgraded PostgreSQL and the database to version 16 and am now getting the error on startup. Using pgAdmin to look at the database does not show any glaring problems. Is there something that I can correct in the database to remove the issue?
Thanks,

**Comment by SirSeba98** [comment]
I had the same issue when I upgraded to the latest version.

In my case I backuped the DB and then deleted the containers and volumes of the DB and Web.
Then I force-recreated both of them and uploaded the backuped DB to the new container and that fixed it.

**Reply by arkansas-coder** [reply]
Thanks for the suggestion. I'll try it and report the results.
